src/gf.c

```diff
@@ -744,36 +744,40 @@ struct ambiguous_matches_env {
     struct typemap_intersection_env match;
     union jl_typemap_t defs;
     jl_typemap_entry_t *newentry;
+    jl_array_t *shadowed;
+    int after;
 };
+const int eager_ambiguity_printing = 0;
```

Comment:
Do we now reliably pick up ambiguous methods `Core`? If not, would it help to turn this on during Travis builds?